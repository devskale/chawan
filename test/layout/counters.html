<!DOCTYPE html>
<!-- Once again, Blink disagrees with Gecko.  This time though, it's
     Blink being weird: it resets list-item counters per div level.
     That feels wrong, so we follow Gecko.  -->
<style>
myli { display: list-item; margin-left: 5ch; list-style-type: decimal }
myli { counter-increment: asdf 1 }
myli::before { content: counter(asdf) ' ' }
:root { counter-set: my-link-counter }
a[href]::before{
  content: '['counter(my-link-counter, cjk-heavenly-stem)']';
  counter-increment: my-link-counter
}
</style>
<a href=https://example.org>test</a>
<div>
  <myli>1</myli>
  <div>
    <a href=https://example.org>test</a>
    <myli>2</myli>
    <myli>3</myli>
    <myli>4</myli>
    <myli>5</myli>
  </div>
  <myli>6</myli>
  <myli>7</myli>
</div>
<div>
  <myli>1</myli>
  <myli>2</myli>
  <myli>3</myli>
  <div>
    <myli>4</myli>
  </div>
  <myli>5</myli>
</div>
<a href=https://example.org>test</a>
