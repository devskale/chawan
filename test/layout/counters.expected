[33m[ç”²]test[39m
  1. 1 1
[33m[ä¹™]test[39m
  2. 2 2
  3. 3 3
  4. 4 4
  5. 5 5
  6. 6 6
  7. 7 7
  1. 1 1
  2. 2 2
  3. 3 3
  4. 4 4
  5. 5 5
[33m[ä¸™]test[39m
